RNode_Firmware.ino

```diff
@@ -935,12 +944,14 @@ void updateModemStatus() {
     portEXIT_CRITICAL(&update_lock);
   #endif
 
-  if (status & SIG_DETECT == SIG_DETECT) { stat_signal_detected = true; } else { stat_signal_detected = false; }
-  if (status & SIG_SYNCED == SIG_SYNCED) { stat_signal_synced = true; } else { stat_signal_synced = false; }
-  if (status & RX_ONGOING == RX_ONGOING) { stat_rx_ongoing = true; } else { stat_rx_ongoing = false; }
+  if ((status & SIG_DETECT) == SIG_DETECT) { stat_signal_detected = true; } else { stat_signal_detected = false; }
+  if ((status & SIG_SYNCED) == SIG_SYNCED) { stat_signal_synced = true; } else { stat_signal_synced = false; }
+  if ((status & RX_ONGOING) == RX_ONGOING) { stat_rx_ongoing = true; } else { stat_rx_ongoing = false; }
```

Comment:
And here I remember myself going on to someone about right-hand associativity...

Well spotted! Functionally, it didn't make much difference, luckily, since SIG_DETECT stays enabled during SIG_SYNCED too, but good to have it correct now.